Cryptomator/Common/CloudAccountList/AccountListViewController.swift

```diff
@@ -127,6 +127,7 @@ class AccountListViewController: SingleSectionTableViewController {
 	}
 
 	override func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
+		tableView.deselectRow(at: indexPath, animated: true)
```

Comment:
Not sure if I'm nitpicky here. Would it somehow be possible to "know" if a deselect is even necessary? Table view cells have this "magic" behavior with a deselect animation when using the swipe gesture to go back a view. This gets lost if the row gets deselected prematurely.

Reply:
I think we should postpone this until we start our general refactoring of the `UITableViewControllers`.
But you're right, we should definitely keep this in mind.